modules/calypsoify/class.jetpack-calypsoify.php

```diff
@@ -274,13 +280,15 @@ public function enqueue_for_gutenberg() {
 		);
 	}
 
-	public function insert_sidebar_html() { ?>
+	public function insert_sidebar_html() { 
+		$heading = ( $_GET[ 'post_type' ] == 'feedback' ) ? 'Feedback' : 'Plugins';
```

Comment:
One should be able to translate this.

```suggestion
		$heading = ( 'feedback' === $_GET['post_type'] ) ? __( 'Feedback', 'jetpack' ) : __( 'Plugins', 'jetpack' );
```